### PR TITLE
Feature/103/general info

### LINKED ISSUE #103: (SP: 2) Implement General Info step
## Description
Implement the general info step to used it in both student and tutor cause they are the same according to the mockup.

## Design 

image

## Acceptance Criteria
- [ ] First name and Last name of a user should be pulled from first registration form.
- [ ] Country and city have to be pulled from server on input focus
- [ ] Additional info length can't be more than 70 symbols

### PR DESCRIPTION
**Pull request by mistercapxx**
#103 - General Info Step for localhost student and tutor

Screenshot 2024-07-26 at 12 30 26

### CODE REVIEW

**Review by markgol777 — CHANGES_REQUESTED** [review]
Too many changes and most of them do not correspond to the task

**Review by Yaroslav-Yakovlev — APPROVED** [review]

**Review by markgol777 — APPROVED** [review]
amazing

**Comment by Yaroslav-Yakovlev** [comment]
The screenshot does not match the layout
Screenshot_3

**Review by Bakalina — APPROVED** [review]

**Comment by mistercapxx** [comment]
Screenshot 2024-08-02 at 14 37 39
made it fine looking

File: src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx [hunk 1]
```diff
@@ -1,12 +1,116 @@
...
 
 const GeneralInfoStep = ({ btnsBox }) => {
+  const [countryList, setCountryList] = useState([])
+  const [cityList, setCityList] = useState([])
+  const [selectedCountry, setSelectedCountry] = useState('')
+  const [selectedCity, setSelectedCity] = useState('')
+  const [professionalStatus, setProfessionalStatus] = useState('')
+  const [charCount, setCharCount] = useState(0)
+
+  const countryCityMap = {
```
Comment (CallMeMaverick) [inline src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx:16]: I think there is a backend task to fetch countries and respective cities #94 
So I believe this logic is redundant 

But generally looks sound!

**Review by Yaroslav-Yakovlev — APPROVED** [review]

**Review by CallMeMaverick — APPROVED** [review]

**Review by kryzanivska-nastya — APPROVED** [review]

**Review by nrevutska — APPROVED** [review]
Create a new task to connect this with backend and implement it.

**Pull request merged**